redis/k8s-redis/loadbalancer.tf

```diff
@@ -0,0 +1,28 @@
+# This resource sets up the redis load balancer service
+resource "kubernetes_service" "loadbalancer" {
```

Comment:
Please be consistent on the convention of naming resources, use `load_balancer` instead
https://www.terraform-best-practices.com/naming